src/Components/Menu/IndexMenu.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { Col, Row } from 'antd';
+import Button from '../Button/Button';
+import './IndexMenu.scss'
+import NavMenu from './NavMenu'
+
+const IndexMenu = ({addProducto, carrito}) => {
+
+    const [tab, setTab] = useState('comida')
+    return ( 
+        
+        <Col className='order' size='large' span={14}>
+            <div className="btnMenu">
```

Comment:
no usar camel case en selectores
```suggestion
            <div className="btn-menu">
```